common/src/main/java/com/wynntils/features/ui/ItemGuideListHotkeyFeature.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.ui;
+
+import com.wynntils.core.config.Category;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.config.ConfigCategory;
+import com.wynntils.core.config.RegisterConfig;
+import com.wynntils.core.features.Feature;
+import com.wynntils.core.features.properties.RegisterKeyBind;
+import com.wynntils.core.keybinds.KeyBind;
+import com.wynntils.screens.base.WynntilsMenuScreenBase;
+import com.wynntils.screens.guides.WynntilsGuidesListScreen;
+import com.wynntils.screens.guides.emeraldpouch.WynntilsEmeraldPouchGuideScreen;
+import com.wynntils.screens.guides.gear.WynntilsItemGuideScreen;
+import com.wynntils.screens.guides.ingredient.WynntilsIngredientGuideScreen;
+import com.wynntils.screens.guides.powder.WynntilsPowderGuideScreen;
+import com.wynntils.utils.mc.KeyboardUtils;
+import org.lwjgl.glfw.GLFW;
+
+@ConfigCategory(Category.INVENTORY)
+public class ItemGuideListHotkeyFeature extends Feature {
+
+    @RegisterKeyBind
+    private final KeyBind itemGuideListKeyBind =
+            new KeyBind("Open Item Guide List", GLFW.GLFW_KEY_UNKNOWN, true, this::onItemGuideListKeyPress);
+
+    @RegisterConfig
+    public final Config<Boolean> emeraldPouchGuide = new Config<>(false);
+
+    @RegisterConfig
+    public final Config<Boolean> powderGuide = new Config<>(false);
+
+    @RegisterConfig
+    public final Config<Boolean> ingredientGuide = new Config<>(true);
+
+    @RegisterConfig
+    public final Config<Boolean> itemGuide = new Config<>(true);
```

Comment:
I don't think there is any need to do this; you can just make a keybind for every guide, and let the keybind configurations differ (or they can use the same key with modifiers)

Reply:
I am not sure why you marked this as complete. Let me phrase it again, maybe I was unclear. Create a keybind for every screen instead of hard-coding modifiers.

Reply:
oh sorry? the (or they can use the same key with modifiers) confused me